com.valvesoftware.Steam.yml

```
@@ -180,13 +180,12 @@ modules:
         sed -i s:Exec=steam:Exec=/app/bin/steam-wrapper: steam.desktop
         sed -i s:/usr/bin/steam:/app/bin/steam-wrapper: steam.desktop
```

Shouldn't this be something like this?
```suggestion
        sed -i s:/usr/bin/steam:/app/bin/steam: steam.desktop
```

---

Also, `command: steam-wrapper` should be `command: steam`.

---

I suppose you're right. @smcv any idea why it uses full path to steam in Desktop Entry?